apps/amiapp_flutter/fastlane/Fastfile

```diff
@@ -1,341 +1,2 @@
-  branch: "main", 
-  path: "fastlane/Fastfile"
-)
-
-# This file contains the fastlane.tools configuration
-# You can find the documentation at https://docs.fastlane.tools
-#
-# For a list of all available actions, check out
-#
-#     https://docs.fastlane.tools/actions
-#
-# For a list of all available plugins, check out
-#
-#     https://docs.fastlane.tools/plugins/available-plugins
-#
-
-# Uncomment the line if you want fastlane to automatically update itself
-# update_fastlane
-
-require 'json'
-
-# Fastlane reacts differently by when it gets executed in GitHub Actions.
-class GitHub
-  # `event` payload for pushes: https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#push
-  # `event` payload for pull requests: https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#pull_request
-
-  def initialize(github_context)
-    @github_context = github_context
-  end
-
-  def is_push
-    @github_context["event"]["commits"] != nil
-  end
-
-  def is_pull_request
-    @github_context["event"]["pull_request"] != nil
-  end
-
-  def commit_hash
-    if is_push
-      return @github_context["event"]["head_commit"]["id"][0..8]
-    else
-      return @github_context["sha"][0..8]
-    end
-  end
-
-  def pr_title
-    @github_context["event"]["pull_request"]["title"]
-  end
-
-  def pr_number
-    @github_context["event"]["pull_request"]["number"]
-  end
-
-  def source_branch
-    @github_context["head_ref"]
-  end
-
-  def destination_branch
-    @github_context["base_ref"]
-  end
-
-  def author
-    if is_push
-      return @github_context["event"]["head_commit"]["committer"]["username"]
-    else
-      return @github_context["event"]["pull_request"]["user"]["login"]
-    end
-  end
-
-  def commit_message
-    @github_context["event"]["head_commit"]["message"]
-  end
-
-  def branch_name
-    @github_context["event"]["ref"].split("/").last # getting the last part of `refs/heads/test-dump` is the branch name
-  end
-end
-
-platform :ios do  
-
-  is_enterprise_app = false
-  info_plist_file_path = "ios/Runner/Info.plist"
-  google_service_plist_file_path = "ios/Runner/GoogleService-Info.plist"
-
-  before_all do |lane, options|
-    if ENV['CI'] 
-      setup_ci 
-
-      # authenticate with apple account so all lanes are able to authenticate correctly
-      # https://docs.fastlane.tools/app-store-connect-api/
-      app_store_connect_api_key(
-        key_id: ENV["APP_STORE_CONNECT_API_KEY_ID"],
-        issuer_id: ENV["APP_STORE_CONNECT_API_ISSUER_ID"],
-        key_content: ENV["APP_STORE_CONNECT_API_KEY_CONTENT_B64"],
-        is_key_content_base64: true,
-        in_house: is_enterprise_app
-      )
-    end
-  end
-
-  # example for main builds: `bundle exec fastlane android deploy_app version:1.0.0"`
-  # example for develoment builds (pull request, push): `bundle exec fastlane deploy_app`  
-  lane :deploy_app do |values| 
```

Comment:
Some of this is removed and some moved and updated in `build_helper.rb` and `version_helper.rb`